Detector/DetFCCeeECalInclined/compact/FCCee_ECalBarrel_thetamodulemerged.xml

```diff
@@ -0,0 +1,159 @@
+    <constant name="EMBarrel_rmax" value="BarCryoECal_rmax-CryoBarrelBack-LArBathThicknessBack"/>
+    <constant name="EMBarrel_dz" value="BarECal_dz-CryoBarrelSide"/>
+    <!-- thickness of active volume between two absorber plates at EMBarrel_rmin, measuring perpendicular to the readout plate -->
+    <constant name="LAr_thickness" value="LArGapThickness"/>
+    <!-- passive layer consists of lead in the middle and steel on the outside, glued -->
+    <!-- When employing trapezoidal planes Pb_thickness corresponds to the minimum thickness, i.e at the front of the calo -->
+    <constant name="Pb_thickness" value="1.80*mm"/>
+    <constant name="planeLength" value="-EMBarrel_rmin*cos(InclinationAngle) + sqrt(EMBarrel_rmax*EMBarrel_rmax - EMBarrel_rmin*EMBarrel_rmin*sin(InclinationAngle)*sin(InclinationAngle))"/>
+    <constant name="ECalBarrelNumPlanes" value="1545"/>
+    <constant name="phi" value="asin(planeLength / EMBarrel_rmax * sin(InclinationAngle))"/>
+    <!-- use a different value for Pb_thickness_max when employing trapezoidal planes -->
+    <!-- approximate constant sampling fraction: make the absorber grow linearly with the radius,
+         taking into account the angular projection effect -->
+    <!-- <constant name="Pb_thickness_max" value="1.3 * Pb_thickness * EMBarrel_rmax/EMBarrel_rmin *
+         cos(InclinationAngle - phi) / cos(InclinationAngle)" />-->
+    <constant name="Pb_thickness_max" value="Pb_thickness" />
+    <!-- total amount of steel in one passive plate: it is divided for the outside layer on top and bottom -->
+    <constant name="Steel_thickness" value="0.1*mm"/>
+    <!-- total amount of glue in one passive plate: it is divided for the outside layer on top and bottom -->
+    <constant name="Glue_thickness" value="0.1*mm"/>
+    <!-- readout in between two absorber plates -->
+    <constant name="readout_thickness" value="1.2*mm"/>
+  </define>
+
+  <display>
+    <vis name="ecal_envelope" r="0.1" g="0.2" b="0.6" alpha="1" showDaughers="false" visible="true" />
+  </display>
+
+  <readouts>
+    <!-- readout for the simulation -->
+    <!-- offset in theta is the eta max value including cryostat TO BE FIXED -->
+    <!-- THIS LACKS Z POSITION -->
+    <!--readout name="ECalBarrelTheta">
+        <segmentation type="GridTheta" grid_size_theta="0.009817477/4" offset_theta="0.5902785"/>
+        <id>system:4,cryo:1,type:3,subtype:3,layer:8,module:11,theta:10</id>
+    </readout-->
+
+    <!-- readout for the reconstruction, with phi-theta -->
+    <!-- readout name="ECalBarrelPhiTheta">
+        <segmentation type="FCCSWGridPhiTheta" grid_size_theta="0.009817477/4" phi_bins="1545" offset_theta="0.58902785" offset_phi="-pi+(pi/1545.)"/>
+        <id>system:4,cryo:1,type:3,subtype:3,layer:8,phi:11,theta:10</id>
+    </readout -->
+
+    <!-- readout for the simulation, with no merged modules and no merged cells in theta -->
```

Comment:
The one without merged module/theta cells is commented out, is this on purpose?

Reply:
while debugging the merging we used as baseline readout the one with no merging in module and theta in any layer, and then merged the cells via RedoSegmentation. Now we have decided to make the default readout the one with the "baseline" merging of 2x in modules in each layer and 4x in theta in all layers but layer 1 (so that the RedoSegmentation is not even needed). Should I remove the "ECalBarrelModuleThetaMerged2" segmentation (which is used for debugging purposes only)? I'd still like to keep a bit the code there (maybe commented) to have an example of how to have two different readouts in output (useful, as I said before, to check that the merging is working as expected)

Reply:
Ok, I see, thanks for adding the comments